### PR TITLE
feat: responsive Ui - page container - side bar - header

### PR DESCRIPTION
**Pull request by domw30**
[Associated Notion Card](https://www.notion.so/zerotech/dApp-Projects-e96437225e264ae8ae8d46ca5f4d7b35?p=b32b5b99549f4d49873b186cb0a0b428)

-----

## 1. Pull request checklist

- [x] Notion card has been moved to the Code Review column
- [x] Notion card has a link to this PR
- [x] A reviewer has been assigned to the Notion card

## 2. PR type
- [x] Feature

## 3. What is the old behaviour?
Not very good responsiveness of the page container

## 4. What is the new behaviour?
Improved responsiveness of the page container

### CODE REVIEW

File: src/containers/PageContainer/index.tsx [hunk 1]
```diff
@@ -93,23 +107,37 @@ const PageContainer: React.FC = ({ children }) => {
...
+								account={account}
+								openModal={openModal}
+								isScrollDetectionDown={isScrollDetectionDown}
+							/>
+
+							{/* Children Components */}
+							<main className={styles.Main}>{children}</main>
+						</div>
+						{/* Actions */}
+						<Actions
```
Comment (domw30) [inline src/containers/PageContainer/index.tsx:132]: Re-added actions here. When reaching laptop/desktop breakpoint in the figma designs, the actions are removed from the header and are required outside of that container...tricky one.

**Comment by domw30** [comment]
@colbr, ready for review if you have time 🙏

File: src/containers/PageContainer/PageContainer.constants.ts [hunk 2]
```diff
@@ -5,4 +5,16 @@ export enum Modal {
 	Wallet = 'Wallet',
 }
 
-export const MODAL_VISIBILITY_WINDOW_MIN_WIDTH = 1000;
+export const MODAL_VISIBILITY_WINDOW_MIN_WIDTH = 744;
```
Comment (domw30) [inline src/containers/PageContainer/PageContainer.constants.ts:8]: Updated to a lower width due to designs requiring 'Connect' button being displayed. Gave this a quick test and Modals still fit within this value width.

File: src/constants/nav.ts [hunk 3]
```diff
@@ -6,7 +6,7 @@ import { ROUTES } from './routes';
 
 export const LINKS = [
 	{
-		label: 'Market',
+		label: 'NFTs',
```
Comment (domw30) [inline src/constants/nav.ts:9]: As per design. Let me know if we want this to go back to Markets

File: src/containers/PageContainer/elements/Touchbar/Touchbar.tsx [hunk 4]
```diff
@@ -0,0 +1,35 @@
+//- React Imports
```
Comment (colbr) [inline src/containers/PageContainer/elements/Touchbar/Touchbar.tsx:1]: Nice, this looks real clean

**Review by colbr — APPROVED** [review]
All looking good to me dude! Great work as per usual

**Pull request merged**